environments/dev.env

```diff
@@ -1,2 +1,2 @@
 NEXT_PUBLIC_APP_NAME=wisee
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8080/v1
+NEXT_PUBLIC_API_BASE_URL=http://localhost:8080
```

Comment:
why localhost is here

Reply:
- these keys are for development environment